tests/system/providers/google/cloud/data_loss_prevention/example_dlp_info_types.py

```diff
@@ -64,12 +65,14 @@
 OBJECT_GCS_OUTPUT_URI = OBJECT_GCS_URI + FILE_NAME
 
 CUSTOM_INFO_TYPE_ID = "custom_info_type"
-CUSTOM_INFO_TYPES = {
-    "large_custom_dictionary": {
-        "output_path": {"path": OBJECT_GCS_OUTPUT_URI},
-        "cloud_storage_file_set": {"url": OBJECT_GCS_URI + "*"},
+CUSTOM_INFO_TYPES = StoredInfoTypeConfig(
+    {
+        "large_custom_dictionary": {
+            "output_path": {"path": OBJECT_GCS_OUTPUT_URI},
+            "cloud_storage_file_set": {"url": OBJECT_GCS_URI + "*"},
```

Comment:
should we use f-string instead?
```suggestion
            "cloud_storage_file_set": {"url": f"{OBJECT_GCS_URI}*"},
```

Reply:
Why not :)